src/main/java/org/gridsuite/sensitivityanalysis/server/service/SensitivityAnalysisService.java

```diff
@@ -116,12 +116,6 @@ private Long getFactorsCount(SensitivityFactorsIdsByGroup factorIds, UUID networ
         return contAttributesCountTemp;
     }
 
-    private Integer getContingenciesCount(List<UUID> ids, UUID networkUuid, String variantId) {
```

Comment:
Do you need to replace this function ?

Reply:
i think this method is not used  so removed it